app/assets/stylesheets/settings.scss

```diff
@@ -41,6 +41,9 @@ $input-width: 650px;
         display: block;
         float: right;
       }
+      .crayons-indicator {
```

Comment:
We want to avoid modifying the CSS classes of our design system even if they are within another element. Instead you can use our margin left utility class `ml-1` which is 4px. We tend to use even numbers for spacing.

So this CSS can be removed, and simply add `ml-1` in the JSX below

`<span className="crayons-indicator crayons-indicator--accent ml-1">`

Or do you think `ml-2` (8px) would be better @ludwiczakpawel? I feel like `ml-1` should be good enough.

We're still building out our design system @anyruizd, so there's some documentation we still need to work on to surface all these utility classes.

Reply:
@nickytonline Thaanks a lot! I was wondering if that was the right approach 😅 
This is how it looks with `ml-1`
<img width="225" alt="imagen" src="https://user-images.githubusercontent.com/29681222/85818270-dbfa8700-b735-11ea-93dc-f8e5b834f6cd.png">
And this with `ml-2`
<img width="252" alt="imagen" src="https://user-images.githubusercontent.com/29681222/85818424-33005c00-b736-11ea-8050-a87eb372ccbb.png">
LMK! 😄

Reply:
I think that looks great, but I'll let @ludwiczakpawel give the final thumbs.

Reply:
Both look great :) I'd probably lean towards ml-2 but that's just personal opinion... You decide :)

Thank you so much @anyruizd for looking into this!